qwiic_i2c/micropython_i2c.py

```diff
@@ -43,8 +43,13 @@
 def _connectToI2CBus(freq=400000):
 	try:
 		from machine import I2C, Pin
-		# Todo: Don't hard code I2C pin and port!
-		return I2C(id=1, scl=Pin(19), sda=Pin(18), freq=freq)
+		if sys.platform == 'rp2':
+			# Todo: Don't hard code I2C pin and port!
+			return I2C(id=1, scl=Pin(19), sda=Pin(18), freq=freq)
+		elif 'xbee' in sys.platform:
+			return I2C(id=1, freq=freq)
```

Comment:
@faludi Does this look like it would work for all XBees? It works with the one I have

Reply:
Yes, it should work with all of them.

Reply:
Great, thank you!